parsers/AR.py

```diff
@@ -37,9 +55,15 @@ def fetch_production(
     non_renewables_production: Dict[str, dict] = non_renewables_production_mix(
         zone_key, current_session
     )
-    renewables_production: Dict[str, dict] = renewables_production_mix(
-        zone_key, current_session
-    )
+
+    if zone_key == "AR":
```

Comment:
I'm thinking that it would be better to just have one `renewables_production_mix` function. The logic for switching to national or regional view could live in there.